src/ProjectTemplates/Web.ProjectTemplates/content/RazorClassLibrary-CSharp/.template.config/vs-2017.3.host.json

```
@@ -12,5 +12,14 @@
   },
   "order": 700,
   "icon": "vs-2017.3/RazorClassLibrary.ico",
-  "learnMoreLink": "https://go.microsoft.com/fwlink/?linkid=872103"
+  "learnMoreLink": "https://go.microsoft.com/fwlink/?linkid=872103",
+  "symbolInfo": [
+    {
+      "id": "SupportPagesAndViews",
+      "name": {
+        "text": "_SupportPagedAndViews"
```

I think this is the text that shows up in VS, correct?

```suggestion
        "text": "_Support pages and views"
```

---

I was operating under the idea that the `_` indicated this was a key rather than a value. [This instance](https://github.com/aspnet/AspNetCore/blob/fb5da88b256c45c19adc3bc81d8f78d6f529e4fb/src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/.template.config.src/vs-2017.3.host.json#L25) would seem to support your formatting, but it's not clear to me why some of these words are underlined and what that means for them.

---

My guess is that `_` indicates the letter for the keyboard shortcut, but @phenning would know for sure.

---

_ indicates the character for the keyboard shortcut like dan said.

---

the text value here is the raw text, not a key to the resource.

---

This looks good, I'll work on getting you updated strings.json files to handle the localization. I just need to do some small tweaks to my loc tool.